docs/source/gbd2019_models/risk_exposures/wasting_states/SAM_compartmental_diagram

```diff
@@ -0,0 +1 @@
1ozEp1RiWX7Jk1zGI6stjMtiMvyPOKYbJXMbAoJjNMnuKzv0zhVJIZyEq7yV55nKhITyurZCpKziyVl5zOoixHOuCZGGSVlPxXJk3dVZZQ+LCSOPMymEg4GVbVl2WJ3JtliSXUsZQUd0wusMmERqXxHHFNFJ8n5TovYGkxYBY5GauLYjaF3C5sx4GaJ5mlJA7YwoqlNArcXD4rVV8dEOlrPirKQnms8nzKoK7xGU9ylXIIj8zdSaedxg/Ewc8GdAre6PSNTt/o9I1OT4hO8xjOqdCpeshWXd6Gk3N2kp9pMfDi2B83dNp3riSl805qr4XsmfvO1RWBOAuYF5GtoZV9Z7UgCLYX1PIGtgWfbyy89pNvbDy/N4DN3/9J310o3i/XfMCnL5vSy2cc+bQXDEjbEdE4LHEiYwShA40R6OgPnnY2Rprcbeg5oMEQvt+h5abmAsjZVVTbBjMVg0VPdWhPcrY89BaJaVQgCtW5UxfZaW/uVCOUdZarjey8CsvlXs7JWE6Nd9ZZrl/Hgg5tOQBPzHL2iV/da/sMJXKUqa+q65qp76k3bpCDtPU0u0GjaWfHd2js0zije7SRYoGuRooFnjNSNO3seKSI5WE5pHiX7jW+Co635UWtCdQzu52GJBzVIT4GcJsdrm8bxWbFOKZN1+lG8c9qhGjNsqd6UHt7uW0jUL2c/DnCD1TZPglTxSfzR0+9t/cyAWkBmQ/F+C8BsmNEqmdgbouzVa/cOsBUXeJuzeOq61CVMHse8rEq25CgcnWoKfHZrr21nJaJzm3ilPbcVIqKEdzPVpYLtxfUtrF0P5nRkxOwg9d5/HXL6fZ+HH+VfxjAtI99/NXVXdc4VWd856aTuA3T+k1XW6ZAG1WKaJm5huqKtvy7ANqzne9Gb78L8DS0gsqeoSUOrx/tdwGabOb3B6ywK7C6khkd0DFYoWK1272g+nYtvSFQbefYQH27W9U353LU/7tVbvW+vxpX6ta5HKqRXv9u9GoC71bFHqbKSp3GkYZN4kin4j60vtFaMY49lALke4Y0lJloe7Ftex66aFRPpqFXOgf1PsAh72fZwvltYQ4a8CPoJcgUh8/R1f8=</diagram><diagram id="61g1iWJjo9wDZMDGOa4t" name="Page-2">7Vrbbts4EP0aA90HFyIl+fJoJ+m2wKYI4Ic2TwFjMTKzsqhKdGzv1y8pkdSFlmWnli91gCIQD8ekeM4MZyi2Y9/MV3/HKJrdUw8HHWh5q45924Gw71r8rwDWGeC4gwzwY+JlkJUDE/IfzkCg0AXxcCKxDGKUBoxEZXBKwxBPWQlDcUyXZbMXGnglIEI+NoDJFAUm+oN4bJahA7UsgX/FxJ+pmYEle+ZIGUsgmSGPLguQfdexb2JKWfY0X93gQHCnePGDX19+rMe0C58YeLwbIceNutlgX/b5iV5CjEP27qG/M5ssH179b93Xx8Ra/Ry/fJ3In1hvKFhIvuRa2VoRiD3Op2zSmM2oT0MU3OXoOKaL0MNiGou3cpt/KI04CDj4ihlbS+dAC0Y5NGPzQPZmc4qJKpo1LFjaJXQRT/GWVcpFMRT7mG2xc7WqPBownWMWr/nvYhwgRt7KL4ekX/raLueeP0j695ACGlJ8p2FXMBuHfIEkmXGKYS/gCxg/x/zJZyl1ElHAFDE+iqNw/irPuW1F2lw4ocJyRhieRCilcsk3g00iveGY4dU7ZDJpXakNJRtG7i896ZLLPFiHEpoV4lSZHVwG+yoiwt0xIoC1WbvjhIS7QYsdA8C+nAAAw8YIGBwzAgYG60SRmSw0l0+c5yLJhS6DZk4YK3OZsJj+i29oQGOOhDQUUfNCgqACoYD4IW9OOcGY42NBP+HZfSQ75sTz0pDbJF45DFvTr3kHczboB9vSb3gVO5iqL5u3sBr1jrOFAbPCuh/d80hBc+Gh4XMSFTexpm0NXs62ZjvwrLY1AAwlJqkSuzEPLod5p2+dF/NmaVuXUWBdRpHWEQpLhPd+LWiqUZY2RiKu/Wf0iS+F/+Ova218+ks8imH5eTJk3Rc0J8E6+/mchjRJJSuZJOk+JwysaJXPm7+gWz1Cu5whgaYHRN1SjLkpZxy5Fc/ixVzBkcuJbrIF2lY5yruGgfkwmTC6h4i/gnCOZAK56TJFU6b9vD+drGKUj5wppUdOCwFhkUaBgEE2SaEcEGia/UVHXhKU4KwsEJAsDARYLg1EpywORGchFEVPGowCVwErQKvy8jIydyFed2nO87B0RWDmlqJaUMOsc7yXaycDVXc5hS4ZsLmKhT4euBr3C9NXHSJtaq8ogmVflXaGU+cRmobjVRZ9egc9k6IPmOfWuj0WfOyxH3vsxx77scee+x7rwDPbY50dTtahNxK3FoLaACUJmR7wWGwyVaDC3UCFwnY+68oZHigRkVd3oACgQnF2Wpe/goU7iOpAau66gbLTvDFQKpde9m8ouOmL4p4K4hVhP4Xnf7ZAT7Yf0/YAqvbtSoZG2lgXGg84JnwVIuQKsdP2Z5JTOY5tH8hxYN89reP0rjT0wRB+bqB+Vw2B3W8aqm0V+4aK8b5F8llmYH013Wo6tirBDM18vOm7Unv52LypqJOz9rvS9cqpy6mzkdO8uKiT87Iuno4iJ4BnJic0o9PQZ7qI3/TX8j8yf9Z8Jto7d1rbx6lJnJxPtC6YRcIgqX/bSp3vWFtfqrI2uN1aJ3tpbpfM+UP2sgdN+HCHu9A/3wf5QAdxwrr/qnNgJ4SgPE2DF1ZX1+CG0C2bH8MN1Snout3Q7sODuKE93D7OgdzQ7pWnaXDD6uoa3NCxyuZHcUPzBvwK3dBtSKW7uqFb83XywG7oDPfKydXVNbihax8/KdvQcMPkaUOhz+t856Lq/E5Lpb2qyk713ds27xbrBLusg1lbgukK5mSKmTcVdYpd1peRthTTyf5kipk3E3WKXdanybYU03nx8Ip15CVtIevl17P23f8=</diagram></mxfile>
```

Comment:
I don't think this file needs to go in the repo. Do let me know if I'm wrong about that, though.